client/my-sites/google-my-business/select-business-type/style.scss

```diff
@@ -59,6 +59,7 @@
 	}
 
 	img {
+		flex: 1;
```

Comment:
This does seem to change to aspect ratio of the image a bit, which makes the text look a little stretched. Looks much better otherwise, though!

Reply:
@scruffian It looks like took care of the img sizing other ways so I don't think `flex: 1` is needed anymore and as @cbauerman points out it stretches the image out.